pkgs/tools/networking/strongswan/default.nix

```diff
@@ -89,15 +99,27 @@ stdenv.mkDerivation rec {
          "--enable-sqlite" ]
     ++ optionals enableNetworkManager [
          "--enable-nm"
-         "--with-nm-ca-dir=/etc/ssl/certs"
+         "--with-nm-ca-dir=/etc/ssl/certs" ]
+    # Taken from: https://wiki.strongswan.org/projects/strongswan/wiki/MacOSX
+    ++ optionals stdenv.isDarwin [
+      "--disable-systemd"
+      "--disable-xauth-pam"
+      "--disable-dependency-tracking"  # from homebrew
```

Comment:
This should be supplied by stdenv by default already

Reply:
Ok, removing. Thanks for catching it.